server/src/main/java/org/elasticsearch/search/aggregations/AggregatorFactories.java

```diff
@@ -75,7 +75,7 @@
             final String aggregationName = parser.currentName();
             if (!validAggMatcher.reset(aggregationName).matches()) {
                 throw new ParsingException(parser.getTokenLocation(), "Invalid aggregation name [" + aggregationName
-                        + "]. Aggregation names must be alpha-numeric and can only contain '_' and '-'");
+                        + "]. Aggregation names must be any character except '[', ']', and '>'");
```

Comment:
Could we make the message say: `Aggregation names can contain any character except '[', ']', and '>'` ?

Reads a little clearner than "must be any character except...".

Reply:
Thx @polyfractal.
I agree, it sounds more cleaner. I will change it and check for test failures

Reply:
I updated the message and run ./gradlew chech on the server module but dont get any test failure.

Reply:
Oops, looks like the new commit erased the CI results.  I'll kick off another build! :)